Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackFindingAlgorithm.hpp

```diff
@@ -97,6 +97,8 @@ class TrackFindingAlgorithm final : public IAlgorithm {
     std::optional<Acts::TrackSelector::Config> trackSelectorCfg = std::nullopt;
     /// Run backward finding
     bool backward = false;
+    /// Maximum number of propagation steps
+    unsigned int maxSteps = 10000000;
```

Comment:
```suggestion
    unsigned int maxSteps = 100000;
```

right I should use the original value here

duing ttbar PU 200 it was actually necessary but maybe there are more bugs hiding